public/index.html

```diff
@@ -14,4 +14,5 @@
     <div id="app"></div>
     <!-- built files will be auto injected -->
   </body>
+  <script src="https://embed.twitch.tv/embed/v1.js"></script>
```

Comment:
more elegant solution would be to only when you will need it, something like here:
https://medium.com/@lassiuosukainen/how-to-include-a-script-tag-on-a-vue-component-fe10940af9e8

Reply:
Yes, looks like it should be done this way.